cargo/cargo.py

```diff
@@ -7,21 +7,21 @@
 }
 
 
-def will_fit(capacity, cargo):
+def will_fit(holds, cargo):
```

Comment:
If you are using python 3, consider using type hints.
So, something like 
```
from typing import List

def will_fit(holds: List[int], cargo: List[int]) -> bool:
    """ Determine if the cargo being loaded will fit with the ship's capacity.

    Args:
        holds:  what is the purpose of this.
        cargo:  what is the purpose of this.
    """
```

Reply:
Oh this is nice - yeah, I remember Josh showing me this and this is awesome. Let me try this out.